src/hooks/useCompareWithPrevious.js

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useRef } from 'react';
+
+const usePrevious = value => {
+    const ref = useRef(value);
+    useEffect(() => {
+        ref.current = value;
+    });
+    return ref.current;
+};
+
+const useCompareWithPrevious = val => {
+    const prevVal = usePrevious(val);
+    return prevVal !== val;
```

Comment:
For å sjekke at en verdi endrer seg kan det kanskje være lettere å angi verdien som en avhengighet i en `useEffect`? Som f.eks.
```javascript
useEffect(() => {
    // Gjør det du trenger å gjøre når behandlingerCtx.state endrer seg her.
}, [behandlingerCtx.state]);
```

Reply:
Jeg tenker at denne:
```javascript
useEffect(() => {
    if (behandlingerChanged && history.location.pathname !== '/') {
        history.push('/');
    }
}, [behandlingerChanged]);
```
kan endres til denne:
```javascript
useEffect(() => {
    if (history.location.pathname !== '/') {
        history.push('/');
    }
}, [behandlingerCtx.state]);
```
Da trenger vi ikke `useCompareWithPrevious.js`